### PR TITLE
Select for countries

### PR DESCRIPTION
**Pull request by kdmnk**
Changed autocomplete to select to ensure a valid country is selected.

### CODE REVIEW

File: resources/views/auth/register/contact.blade.php [hunk 1]
```diff
@@ -1,10 +1,27 @@
...
+            <option value="{{ $country }}" @if($country=="Hungary") selected @endif>{{ $country }} </option>
+            @endforeach
+          </select>
+          <label for="country">@lang('info.country')</label>
+          <script>
+            var instances;
+            $(document).ready(
+              function() {
+                var elems = $('#country');
+                const options = [
```
Comment (luksan47) [inline resources/views/auth/register/contact.blade.php:14]: Is this option really necessary?

File: resources/js/register_helper.js [hunk 2]
```diff
@@ -1,7 +1,4 @@
 $(document).ready(function() {
```
Comment (luksan47) [inline resources/js/register_helper.js:1]: BTW, I think this file should be renamed.

Comment (kdmnk) [reply resources/js/register_helper.js:1]: Actually now it only contains the datepicker initializer (8 lines). we can have them in the blade, I think.

File: resources/views/auth/register/contact.blade.php (hunk 1 above)
Comment (kdmnk) [reply resources/views/auth/register/contact.blade.php:14]: I think so.

File: resources/js/register_helper.js (hunk 2 above)
Comment (luksan47) [reply resources/js/register_helper.js]: Sure.

File: resources/views/auth/register/personal.blade.php [hunk 3]
```diff
@@ -21,6 +21,16 @@
         @error('date_of_birth')
         <span class="helper-text" data-error="{{ $message }}"></span>
         @enderror
+        <script>
```
Comment (luksan47) [inline resources/views/auth/register/personal.blade.php:24]: Do not forget to remove file from the mix compiler as well (webpack.mix.js). The build is failing now.

**Comment by luksan47** [comment]
Could you open a Jira, so we can track the work done?

**Pull request merged**